var/spack/repos/builtin/packages/ioapi/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+import os
+
+from spack import *
+
+
+class Ioapi(MakefilePackage):
+    """Models-3/EDSS Input/Output Applications Programming Interface."""
+
+    homepage = "https://www.cmascenter.org/ioapi/"
+    url      = "https://www.cmascenter.org/ioapi/download/ioapi-3.2.tar.gz"
+    version('3.2', sha256=('56771ff0053d47f2445e00ff369bca7b'
+                           'fc484325a2816b2c648744e523134fe9'))
+    depends_on('netcdf@4:')
+    depends_on('netcdf-fortran@4:')
+    depends_on('sed', type='build')
+
+    def edit(self, spec, prefix):
+        # No default Makefile bundled; edit the template.
+        os.symlink('Makefile.template', 'Makefile')
+        # The makefile uses stubborn assignments of = instead of ?= so
+        # edit the makefile instead of using environmental variables.
+        makefile = FileFilter('Makefile')
+        makefile.filter('^BASEDIR.*', 'BASEDIR = ' + self.build_directory)
+        makefile.filter('^INSTALL.*', 'INSTALL = ' + prefix)
+        makefile.filter('^BININST.*', 'BININST = ' + prefix.bin)
+        makefile.filter('^LIBINST.*', 'LIBINST = ' + prefix.lib)
+
+    def build(self, spec, prefix):
+        make()
```

Comment:
This is the default, no need to override `build`.

Reply:
Fixed in 1060d09